modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/endpoint/dynamic/registration/LiferayDynamicRegistrationService.java

```diff
@@ -270,7 +273,8 @@ private void _checkValidGrantResponseTypes(
 			!allowedResponseTypeList.isEmpty()) {
 
 			OAuthError oAuthError = new OAuthError(
-				"invalid_client_metadata",
+				OAuth2ProviderRESTEndpointConstants.
+					ERROR_KEY_INVALID_CLIENT_METADATA,
```

Comment:
Why not adding it to the `OAuthConstants` class?

Reply:
The OAuthConstans belongs to CXF library

Reply:
this string is indicated in the RFC: https://datatracker.ietf.org/doc/html/rfc7591#section-3.2.2,